lib/omnibus/fetchers/net_fetcher.rb

```diff
@@ -125,6 +125,11 @@ def checksum
       source[digest_type]
     end
 
+    # Set to false to extract archives into the root source_dir (default).
+    # Set to true to always extract underneath project_dir
+    #
+    attr_accessor :isolate_extraction
```

Comment:
@ksubrama I believe that example is bad since the software definition could have been written in such a way to use `relative_path`

So this:

```
relative_path 'libzmq4x-windows`

build do
  copy "#{project_dir}/bin/*", "#{install_dir}/embedded/bin"
  copy "#{project_dir}/include/*", "#{install_dir}/embedded/include"
  copy "#{project_dir}/lib/*", "#{install_dir}/embedded/lib"

  # Ensure the main DLL is available under a well known name.
  copy "#{project_dir}/bin/libzmq-mt-4_0_6.dll", "#{install_dir}/embedded/bin/libzmq.dll"
end
```
instead of
```
build do
  tmpdir = File.join(Omnibus::Config.source_dir, "libzmq4x-windows")

  copy "#{tmpdir}/bin/*", "#{install_dir}/embedded/bin"
  copy "#{tmpdir}/include/*", "#{install_dir}/embedded/include"
  copy "#{tmpdir}/lib/*", "#{install_dir}/embedded/lib"

  # Ensure the main DLL is available under a well known name.
  copy "#{tmpdir}/bin/libzmq-mt-4_0_6.dll", "#{install_dir}/embedded/bin/libzmq.dll"
end
```

Regardless I don't think we should create new config options for 1 or 2 cases we can easily fix.